AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, queue_info, app):
         self.task_retry_limit = retry_params['task_retry_limit']
 
     self.validate_config()
+    self.prepared_statements = {}
```

Comment:
It usually lives forever. However, it should have at most about 10 items in it since prepared statements do not have values bound to them yet.